Source/main.cpp

```diff
@@ -1,10 +1,38 @@
 #include <QApplication>
+#include <QCommandLineParser>
 
 #include "GUI/MainWindow.h"
 
 int main(int argc, char** argv)
 {
   QApplication app(argc, argv);
+  QApplication::setApplicationName("Dolphin Memory Engine");
+  QApplication::setApplicationVersion("0.9.0");
```

Comment:
Lets set this to 1.0.0 as that is the next release.

Please also add the help/args to the README file.